scripts/ci/lib.sh

```diff
@@ -152,6 +152,24 @@ get_central_debug_dump() {
 
     roxctl -e "${API_ENDPOINT}" -p "${ROX_PASSWORD}" --insecure-skip-tls-verify central debug dump --output-dir "${output_dir}"
     ls -l "${output_dir}"
+
+    local metrics_output
+    local csv_output
+    metrics_output="$(mktemp --suffix=.prom)"
+    csv_output="$(mktemp --suffix=.csv)"
+    unzip -p "${output_dir}/*.zip" metrics-1 > "${metrics_output}"
+
+    go install github.com/stackrox/prometheus-metric-parser@latest
+    prometheus-metric-parser single --file /tmp/metrics \
+      | awk -F"," \
+      -v build_tag="${STACKROX_BUILD_TAG}" \
+      -v build_id="${BUILD_ID}" \
+      -v orchestrator="${ORCHESTRATOR_FLAVOR}" \
+      -v job_name="${JOB_NAME:-missing}" \
+      'BEGIN { OFS = "," } {$4=build_tag;$5=build_id;$6=orchestrator;$7=job_name; print}' > "${csv_output}"
+
+    setup_gcp
+    save_central_metrics metrics.csv
```

Comment:
What's `metrics.csv`? Should this have been `$csv_output`?

Reply:
Good catch